BitwardenShared/Core/Vault/Repositories/VaultRepository.swift

```diff
@@ -908,19 +900,16 @@ class DefaultVaultRepository { // swiftlint:disable:this type_body_length
         let typesIdentityCount = activeCiphers.lazy.filter { $0.type == .identity }.count
         let typesLoginCount = activeCiphers.lazy.filter { $0.type == .login }.count
         let typesSecureNoteCount = activeCiphers.lazy.filter { $0.type == .secureNote }.count
+        let typesSshKeyCount = activeCiphers.lazy.filter { $0.type == .sshKey }.count
```

Comment:
⛏️ Format on `Ssh` to `SSH` to follow swift guidelines:
```suggestion
        let typesSSHKeyCount = activeCiphers.lazy.filter { $0.type == .sshKey }.count
```

Reply:
Fixed in 2693f01